paradise_papers_django/search/models/helpers.py

```diff
@@ -0,0 +1,73 @@
+from neomodel import db
+from . import (
+    Entity,
+    Address,
+    Intermediary,
+    Officer,
+    Other
+)
+
+MODELENTITIES = {
+'entity': Entity.Entity,
```

Comment:
Need to find a better way to import this 🤔

Reply:
I think we should drop it... 
I think it is simpler to just write a longer import statement. Also rather consider to import the models into the models/__init__.py to keep the import statement short in other modules.